eslint.config.mjs

```diff
@@ -435,6 +434,11 @@ export default typescript.config([
       '@typescript-eslint/consistent-type-definitions': 'off', // TODO(ryan953): Fix violations and delete this line
       '@typescript-eslint/no-empty-function': 'off', // TODO(ryan953): Fix violations and delete this line
 
+      // migrated from biome
```

Comment:
```suggestion
```

Reply:
should put the added rules into the existing list and keep the list sorted. 

could be fine to make a new list of rules that depend on types, their slowness makes them special.

Reply:
Fixed in a04c43e27be